src/seedsigner/models/encode_qr.py

```diff
@@ -344,7 +344,7 @@ def derivation_to_keypath(path: str) -> list:
             return Keypath(arr, self.root.my_fingerprint, len(arr))
             
         origin = derivation_to_keypath(self.derivation)
-        self.use_info = None if self.network == "M" else CoinInfo(type=None, network=1)
+        self.use_info = None if self.network == SettingsConstants.MAINNET else CoinInfo(type=None, network=1)
```

Comment:
Implements "use_info.network" member provided on HDKey/CoinInfo classes (urtypes/crypto packages-libs folder) for enable testnet public key export option ("self.use_info" assignation for to be "None" when mainnet). #582

Reply:
Replaces "M" for setting constant relative to Mainnet.